src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -39,17 +40,26 @@
 /* controls use of locks to enforce safe commutativity */
 bool AllModificationsCommutative = false;
 
+static uint64 shardForTransaction = INVALID_SHARD_ID;
+static bool callbackInstalled = false;
+static bool txnStarted = false;
```

Comment:
I think I meant it as _local transaction is in progress_, but I'll consider some sort of name change.